hudi-client/src/main/java/org/apache/hudi/table/HoodieCopyOnWriteTable.java

```diff
@@ -299,33 +297,4 @@ public String toString() {
       return sb.toString();
     }
   }
-
-  /**
-   * Obtains the average record size based on records written during previous commits. Used for estimating how many
-   * records pack into one file.
-   */
-  protected static long averageBytesPerRecord(HoodieTimeline commitTimeline, int defaultRecordSizeEstimate) {
```

Comment:
This method seems only being used in a unit test and has an identical copy on upsertPartitioner. So I removed this.